src/views/month/single-day.php

```diff
@@ -40,7 +40,7 @@
 	<div class="tribe-events-viewmore">
 		<?php
 
-			$view_all_label = sprintf( _n( 'View 1 %1$s', 'View All %2$s %3$s', $day['total_events'], 'tribe-events-calendar' ), $events_label_singular, $day['total_events'], $events_label_plural );
+			$view_all_label = sprintf( _n( 'View 1 %s', 'View All %s %s', $day['total_events'], 'tribe-events-calendar' ), $events_label_singular, $day['total_events'], $events_label_plural );
```

Comment:
This [Stackoverflow answer](http://stackoverflow.com/a/27036053) gave a clue, I think. Not all languages handle plurals the same so the same variables need to be in both strings. What if we do the following:

Up above (on lines 16 - 19):
```php
// Change:
$events_label_singular = tribe_get_event_label_singular();
$events_label_plural = tribe_get_event_label_plural();

$day = tribe_events_get_current_month_day();

// To:
$day = tribe_events_get_current_month_day();
$events_label = ( 1 === $day['total_events'] ) ? tribe_get_event_label_singular() : tribe_get_event_label_plural();
```

Then update the line in question with:
```php
$view_all_label = sprintf(
  _n(
    'View %s %s',
    'View All %s %s',
    $day['total_events']
  ),
  $day['total_events'],
  $events_label
);
```